mdb-realm/README.md

```diff
@@ -58,17 +55,15 @@ The action takes the following parameters:
 1. *(Required)* `projectId`: the Id of the Atlas project where the cluster will be created.
 1. *(Required)* `apiKey`: the public [Atlas API key](https://docs.atlas.mongodb.com/configure-api-access/).
 1. *(Required)* `privateApiKey`: the private [Atlas API key](https://docs.atlas.mongodb.com/configure-api-access/).
-1. *(Optional)* `differentiator`: a string to differentiate this deployment from others in the same workflow. This value will be combined with the Github Actions run id to generate a unique string for each run+target combination. It is not required when a specific cluster name is passed.
-1. *(Optional)* `appsPath`: the path where the exported apps are located.
-1. *(Optional)* `atlasUrl`: the Atlas URL to deploy against. Default is https://cloud-dev.mongodb.com.
-1. *(Optional)* `realmUrl`: the MongoDB Realm URL to deploy against. Default is https://realm-dev.mongodb.com.
+1. *(Optional)* `atlasUrl`: the Atlas URL to deploy against. Default is https://cloud-qa.mongodb.com.
+1. *(Optional)* `realmUrl`: the MongoDB Realm URL to deploy against. Default is https://realm-qa.mongodb.com.
 1. *(Optional)* `clusterName`: the name of the cluster that to be created. If it is not set it is auto-generated.
```

Comment:
Explain what the auto-generated name is based on (run id, workflow id, step id, etc)

Reply:
```suggestion
1. *(Optional)* `clusterName`: the name of the cluster that to be created. If it is not set, it is auto-generated and unique per `GITHUB_RUN_ID` and `GITHUB_RUN_ATTEMPT`. 
```